main.tf

```diff
@@ -118,6 +118,16 @@ resource "google_container_cluster" "primary" {
     provider = "CALICO"
   }
 
+  dynamic "resource_usage_export_config" {
+    for_each = var.enable_gke_metered_billing ? ["placeholder"] : []
+    content {
+      enable_network_egress_metering = false
```

Comment:
Do we need Network Egress data @schnie  ?

Reply:
Any reason to not have it? Would be interesting to see it and how it factors in. I enabled in in dev.

Reply:
The only reason I didn't enable it was because we don't charge or at least are not planning to charge users based on network egress. But I will enable it.